### ISSUE TITLE
Grain SSDs does not seem to be reliable on Windows

### ISSUE THREAD

**Issue by adamsewell** [saltstack/salt#47635]
### Description of Issue/Question
In some testing I'm doing, it does not seem that the SSDs grain is reliable for testing for the existence of SSDs. For example, I have a Windows machine with a Samsung Evo 840 SSD which has it's grains.get 'SSDs' returning blank. 

On the one machine that I do have that returns correctly, the model of the drive in WMI is "NVMe SAMSUNG MZVLW512"

### Steps to Reproduce Issue
salt minion grains.get 'SSDs'

### Versions Report
```Salt Version:
           Salt: 2018.3.0
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Dec  4 2017, 14:50:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: Ubuntu 16.04 xenial
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-124-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by Ch3LL** [comment]
on the windows host that is returning blank can you run `salt-call --local grains.get SSDs -ltrace`. It looks like in the code here: https://github.com/saltstack/salt/blob/v2018.3.0/salt/grains/disks.py#L165  it is running the command `wmic /namespace:\\root\microsoft\windows\storage path MSFT_PhysicalDisk where (MediaType=3 or MediaType=4) get DeviceID,MediaType /format:table` 

If you log with trace looks like we migth see why its not returning. you might be running into this:

```
log.trace('Disk grain does not support this version of Windows')
```

**Comment by adamsewell** [comment]
You are correct, this is Windows 7. Is this only supported on Windows 10?

**Comment by Ch3LL** [comment]
looks like @morganwillcock added this feature. Can you speak to the windows os support here?

**Comment by Ch3LL** [comment]
thanks for clarifying @morganwillcock 

@adamsewell are you okay to close as its expected behavior?

**Comment by morganwillcock** [comment]
It looks like you can do it with ATA commands:
[https://stackoverflow.com/questions/23363115/detecting-ssd-in-windows](https://stackoverflow.com/questions/23363115/detecting-ssd-in-windows)

I don't have a Windows 7 device to test it on at the moment.
@Ch3LL, perhaps this can be left here as a feature request?

**Comment by Ch3LL** [comment]
Yep will add as a feature request :)

**Comment by hahanos** [comment]
Hello, 
When i execute grains.ls on osversion:10.0.26100 (24H2), keys ssds, disks are not available. 
Keys ssds, disks are available on osversion:10.0.22631 (23H2).
